### ISSUE TITLE
App admin

### ISSUE THREAD

**Issue by minaorangina** [DRDD2016/native#160]
This issue is to list things like TestFlight and env variable management for production version of app

Related as well #92

**Comment by minaorangina** [comment]
### Dealing with env variables
[This article](http://www.multunus.com/blog/2016/06/automated-environment-management-react-native-ios/) is a step-by-step guide on configuring app env variables to function locally and in production (things like swapping `localhost` url for production one, etc)

**Comment by minaorangina** [comment]
## [React Native Config](https://github.com/luggit/react-native-config)
Module for handling your environment variables.

### iOS setup
```bash 
npm install --save react-native-config
```
```bash 
react-native link react-native-config
```
If the project is already running, rebuild and restart.

**Comment by heron2014** [comment]
Resolved by #162
